packages/components/src/utils/misc-variable-values.js

```diff
@@ -0,0 +1,6 @@
+export default {
```

Comment:
This was bound to happen 😂 .

Here's how I've organized it in G2. It may spark some ideas!
https://github.com/ItsJonQ/g2/blob/master/packages/styles/src/theme/config.js

I think naming it something like `config.js` would be helpful. At the very least, it makes it feel more important/official (which it should be).

Since it's a JS object, you can group certain values together and `...spread` them in the final object :).
That's how I've been keeping things tidy!

Lemme know what you think :D

Reply:
Do you think we should group _everything_ together like G2 does, moving `font` and `config` together into a single object? I'm not necessarily opposed to that 🤷‍♀️ I don't have any strong opinions here so just let me know what you think would be best (I've already renamed this to `config` rather than `misc-variables`).

Reply:
From my experience, when you start having LOTS of variables, it's quite nice to have it somewhat centralized.

It being a flat object vs. nested is subjective, although I've found a single flat objected being easier to work with.

We don't necessarily have to do this to start :). Just wanted to share some thoughts!
The current setup works okay 👍 .

I especially like how you're developing a pattern around accessing values via some sort of function.
It's a workflow that's shared between `breakpoint()`, `font()`, `color()`, and now `config()` :)